src/hotspot/cpu/ppc/sharedRuntime_ppc.cpp

```diff
@@ -47,7 +47,9 @@
 #include "opto/runtime.hpp"
 #endif
 
+#ifndef __FreeBSD__
```

Comment:
None of the BSD have alloca.h from what I can tell.  Please make this _ALLBSD_SOURCE

Reply:
ok